frontend/src/metabase/common/components/QueryColumnPicker/QueryColumnPicker.tsx

```diff
@@ -193,26 +294,35 @@ export function QueryColumnPicker({
   );
 
   const renderItemIcon = useCallback(
-    (item: ColumnListItem) => (
-      <QueryColumnInfoIcon
-        query={query}
-        stageIndex={stageIndex}
-        column={item.column}
-        position="top-start"
-      />
-    ),
+    (item: Item) =>
+      item.type === "expression-clause" ? (
```

Comment:
Do we plan to also support searching for expressions in dropdowns for filters/aggregations/custom columns?